docs/tutorials/Astro/deploying-hybrid-projects.md

```diff
@@ -0,0 +1,286 @@
+    // In production, this should be set via the APOS_HOST env variable
+    aposHost: process.env.APOS_HOST || 'http://localhost:3000',
+    widgetsMapping: './src/widgets',
+    templatesMapping: './src/templates',
+    
+    // Security headers to pass from backend to frontend
+    includeResponseHeaders: [
+      'content-security-policy',
+      'strict-transport-security',
+      'x-frame-options',
+      'referrer-policy',
+      'cache-control'
+    ],
+    
+    // For split deployment (separate servers), you may need to uncomment:
+    // excludeRequestHeaders: ['host']
+  })],
+  
+  // Required to handle virtual URLs in the integration
+  vite: {
+    css: {
+      preprocessorOptions: {
+        scss: {
+          quietDeps: true
+        }
+      }
+    },
+    ssr: {
+      noExternal: ['@apostrophecms/apostrophe-astro']
+    }
+  },
+  
+  // SCSS configuration if using Sass
+  css: {
+    preprocessorOptions: {
+      scss: {
+        api: 'modern-compiler',
+      }
+    }
+  }
+});
+```
+
+### Key Configuration Areas
+
+1. **Port Configuration**
+   - The `server.port` setting defaults to 4321 but reads from the `PORT` environment variable if set
+   - Some platforms (Heroku, Railway) require `host: true` to listen on all interfaces
+
+2. **Integration Settings**
+   - `aposHost` must point to your production backend URL in production
+   - Set via the `APOS_HOST` environment variable rather than hardcoding
+
+3. **Header Configuration**
+   - `includeResponseHeaders` determines which response headers from ApostropheCMS are preserved
+   - Essential for maintaining security settings between backend and frontend
+
+4. **Split Deployment Settings**
+   - When deploying to separate servers, you may need to exclude the `host` header
+   - Uncomment `excludeRequestHeaders: ['host']` to prevent hostname conflicts
+
+
+## Deploying with Apostrophe Hosting
+
+Apostrophe offers a straightforward hosting solution specifically designed for ApostropheCMS projects, including those with Astro frontends.
+
+### Benefits
+
+- Zero configuration for ApostropheCMS + Astro integration
+- Database provisioning and management handled automatically
+- Built-in asset storage and delivery
+- SSL certificate management
+- Automatic backups
+- Security updates
+
+### Getting Started with Apostrophe Hosting
+
+Contact the Apostrophe team through their [website](https://apostrophecms.com/contact-us) to set up your hosting. The team will provide detailed instructions for connecting your repository for automatic deployment.
+
+## Split Deployment (Separate Backend and Frontend)
+
+For more control or to leverage specific platform features, you can deploy the backend and frontend separately.
+
+### Backend (ApostropheCMS) Deployment
+
+Your ApostropheCMS backend requires:
+
+- Node.js environment (v18 or later recommended)
+- MongoDB database connection
+- Asset storage solution (S3 or equivalent cloud storage)
+
+#### Common Backend Hosting Options
```

Comment:
I would put our managed hosting right on this list.

Reply:
Done.